extras/plot_seeding_performance/README.md

```diff
@@ -0,0 +1,30 @@
+# Seeding performance plotter
+
+This Python script plots the performance of seeding code if the
+`nseed_performance_writer` class is used to analyse it. The script takes a file
+of seeds and a file of tracks (with default arguments) and optional filter and
+output flags. For example, simply calling the script should work fine in most
+situations (if the CSV files are in the current directory):
+
+```sh
+$ python extras/plot_seeding_performance/plot.py
```

Comment:
I like that you use `python3` explicitly in the shebang in the script. But then let's be consistent, and use `python3` in the README as well. :wink: